tests/dashboards/api_tests.py

```diff
@@ -1147,7 +1152,12 @@ def test_export_bundle(self):
         """
         Dashboard API: Test dashboard export
         """
-        argument = [1, 2]
+        dashes = (
```

Comment:
nit: would be nice to refactor and make this DRY

Reply:
done